.gitignore

```diff
@@ -23,3 +23,45 @@ SparkR-pkg.Rproj
 # Eclipse Meta Files
 .project
 .classpath
+<<<<<<< HEAD
```

Comment:
This file didn't merge correctly. Actually we don't need updates to gitignore here, so you can revert back to the version we had